conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -26,9 +26,12 @@ show_channel_urls: true
 CONDARC
 )
 
+HOST_USER_ID=$( hash docker-machine 2>/dev/null && docker-machine ssh $(docker-machine active) 'id -u $(whoami)' || id -u $(whoami) )
```

Comment:
> It would also be handy to still be able to run as root. Have these changes prevented that from happening. How would we go about doing that? 

@pelson you can still run as root if you bypass the entrypoint:

```docker run -i -t --entrypoint /bin/bash condaforge/linux-anvil```